opendbc/car/torque_data/override.toml

```diff
@@ -86,3 +86,6 @@ legend = ["LAT_ACCEL_FACTOR", "MAX_LAT_ACCEL_MEASURED", "FRICTION"]
 "HONDA_CIVIC_2022" = [2.5, 1.2, 0.15]
 "HONDA_HRV_3G" = [2.5, 1.2, 0.2]
 "HONDA_PILOT_4G" = [1.0, 1.0, 0.2]
+
+# Hyundai/Kia/Genesis angle control (do we really need this?)
+"HYUNDAI_IONIQ_5_PE" = [3.172929, 3.5, 0.096019]
```

Comment:
are these numbers real? they seem way too high

Reply:
be super careful with these new hyundais. their stock systems will exceed the ISO11270 limits, and we definitely don't want to do that in openpilot

Reply:
The numbers were taken from params.toml for `ioniq_5` except the `MAX_LAT_ACCEL_MEASURED` which I got by seeing the max lat the stock system was doing but let's revisit this, I see all angle vehicles have NaN here so i wonder if I should even put values here or not